src/spikeinterface/core/baserecording.py

```diff
@@ -766,20 +766,20 @@ def get_num_samples(self) -> int:
 
     def get_traces(
         self,
-        start_frame: Union[int, None] = None,
-        end_frame: Union[int, None] = None,
-        channel_indices: Union[List, None] = None,
+        start_frame: int | None = None,
+        end_frame: int | None = None,
+        channel_indices: list | np.array | None = None,
     ) -> np.ndarray:
         """
         Return the raw traces, optionally for a subset of samples and/or channels
 
         Parameters
         ----------
-        start_frame: Union[int, None], default: None
+        start_frame: int | None, default: None
             start sample index, or zero if None
-        end_frame: Union[int, None], default: None
+        end_frame: int | None, default: None
             end_sample, or number of samples if None
-        channel_indices: Union[List, None], default: None
+        channel_indices: list | np.array | None, default: None
             Indices of channels to return, or all channels if None
         order: list or None, default: None
             The memory order of the returned array.
```

Comment:
Why is this here? It is an argument? Was it just copied from the `get_traces` above?

Reply:
I guess it was a bad and lazy copy/paste

Reply:
Cool. I removed it.